pkg/maps/lbmap/lbmap.go

```diff
@@ -494,24 +498,15 @@ func updateServiceV2Locked(fe ServiceKey, backends serviceValueMap,
 	return nil
 }
 
-func removeBackendsLocked(removedBackendIDs []loadbalancer.BackendID, isIPv6 bool,
+func removeBackendsLocked(removedBackendIDs []BackendKey, isIPv6 bool,
```

Comment:
`isIPv6` is no longer needed. Can be addressed in a separate PR.

Reply:
I need to rebase anyway, so I can follow up on this here.